src/storybook-addon.js

```diff
@@ -0,0 +1,134 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-env browser */
+import { toId } from '@storybook/router/utils';
+import deprecate from 'util-deprecate';
+import { stripIndents } from 'common-tags';
+
+import isChromatic from './isChromatic';
+
+const CHROMATIC_PARAMETERS = [
```

Comment:
Is this duplicative? Seems odd

Reply:
it is, I didn't want to go down the path of sharing code between the bin and the addon. I could address this in V4